### DISCUSSION TITLE
Adding animation inside GrapesJS builder

### DISCUSSION THREAD

**Discussion by lifeofsa** [GrapesJS/grapesjs#5415] [Q&A]
How can I add animation to a particular div, i.e a pulse or bounce animation?

**Comment by lifeofsa** [comment]
whenever I want to add this " @keyframes pulse {
  0% {
    transform: scale(0.85);
  }
  50% {
    transform: scale(1);
  }
  100% {
    transform: scale(0.85);
  }
} "
inside my <style></style> tag

GrapesJS returns it with this
 " @media pulse{
  0%{
    transform:scale(0.85);
  }
  50%{
    transform:scale(1);
  }
  100%{
    transform:scale(0.85);
  }
  0%{
    transform:scale(1);
  }
  50%{
    transform:scale(1.1);
  }
  100%{
    transform:scale(1);
  }
} "

What should I do ?

**Reply by artf** [reply]
please provide a reproducible demo on jsfiddle?

**Reply by lifeofsa** [reply]
I understand it. when try to store the data of a template on  a remote server the animation is not working and do the following thing that I mentioned above, however when I store it locally on a local storage then it is working just fine. now what should I do ?
